### PR TITLE
Docs/supported amarel

### PR DESCRIPTION
**Pull request by mtitov**

### CODE REVIEW

File: docs/source/supported/amarel.rst [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+--------------------------
+
+Create a **virtual environment with** ``venv``:
+
+.. code-block:: bash
+
+   export PYTHONNOUSERSITE=True
+   module load python
+   python3 -m venv ve.rp
+   source ve.rp/bin/activate
```
Comment (mtitov) [inline docs/source/supported/amarel.rst:83]: I'm not sure about this part, @AymenFJA can you please check/test it?

**Comment by AymenFJA** [comment]
@mtitov @mturilli @andre-merzky, would we care to add a note in the tutorial regarding that the user needs to enable the VPN in order to access Amarel?

File: docs/source/supported/amarel.rst (hunk 1 above)
Comment (mtitov) [reply docs/source/supported/amarel.rst]: Thank you, Aymen! updated accordingly (and added other option using your explored results)

**Comment by mturilli** [comment]
> @mtitov @mturilli @andre-merzky, would we care to add a note in the tutorial regarding that the user needs to enable the VPN in order to access Amarel?

Why not, maybe as a `.. note::` at the beginning when referencing the user guide?

**Review by mturilli — APPROVED** [review]
LGTM!

**Pull request merged**